src/longsequences/longsequence.jl

```diff
@@ -84,25 +84,29 @@ end
 # Replace a LongSequence's data with a copy, copying only what's needed.
 # The user should never need to call this, as it has no outward effect on the
 # sequence.
-function orphan!(seq::LongSequence{A},
+function orphan!(seq::LongSequence,
 		 size::Integer = length(seq),
-		 force::Bool = false) where {A}
+		 force::Bool = false)
+	if !seq.shared & !force
```

Comment:
Does && introduce branching vs &?

Reply:
Yes, `a && b` guarantees that b will not be evaluated unless a (I guess unless the compiler can prove that evaluation a has no side effects).